### PR TITLE
github: Add tablegen directories to PR autolabeler

### PR DESCRIPTION
**Pull request by arsenm**

### CODE REVIEW

**Comment by arsenm** [comment]
* **#109129** <a href="https://app.graphite.dev/github/pr/llvm/llvm-project/109129?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @arsenm and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: .github/new-prs-labeler.yml [hunk 1]
```diff
@@ -1008,3 +1008,8 @@ bazel:
 
 offload:
   - offload/**
+
+tablegen:
+  - llvm/include/TableGen/**
+  - llvm/lib/TableGen/**
+  - llvm/utils/TableGen/**
```
Comment (boomanaiden154) [inline .github/new-prs-labeler.yml:1015]: Do you want to include `llvm/test/TableGen` and `llvm/unittests/TableGen`?

**Review by boomanaiden154 — APPROVED** [review]
LGTM. You'll need to wait for @tstellar to create the `pr-subscribers-tablegen` team before merging this though.

**Comment by arsenm** [comment]
> LGTM. You'll need to wait for @tstellar to create the `pr-subscribers-tablegen` team before merging this though.

I thought the plan was to get rid of the pr-subscribers lists. I removed myself from all of those a while ago

**Comment by boomanaiden154** [comment]
> I thought the plan was to get rid of the pr-subscribers lists. I removed myself from all of those a while ago

Maybe eventually if Github lets you do label-based subscriptions for PRs, but I believe they only support that for issues currently. That's still how notifications are done currently as far as I can tell.

**Comment by asl** [comment]
> Maybe eventually if Github lets you do label-based subscriptions for PRs, but I believe they only support that for issues currently. That's still how notifications are done currently as far as I can tell.

Yeah. Sadly, PRs are for some reason unknown to me are excluded from label-based subscriptions and notifications.

**Review by asl — APPROVED** [review]

**Comment by tstellar** [comment]
The team is created.

**Pull request merged**